### DISCUSSION TITLE
Is there a way to check if the bot has disconnected / kicked from vc?

### DISCUSSION THREAD

**Discussion by EpicBirb** [nextcord/nextcord#402] [Q&A] [answered]
I've been scratching my head on this, anyone knows how to do this?

**Comment by ooliver1** [comment, marked as answer]
on_voice_state_update may be an event you want

can compare between the two what changed and what user

**Reply by EpicBirb** [reply]
ima try that real quick brb

**Reply by EpicBirb** [reply]
it works, tysm
